### PR TITLE
Adding local flavor for Venezuela

### PR DESCRIPTION
**Pull request by jackboot7**
Added local flavor for Venezuela data. 

- Added regions choice data.
- Added Select widget for regions.

### CODE REVIEW

File: localflavor/ve/ve_region.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+"""
+A list of Venezuelan regions for use as `choices` in a formfield.
+
+Data based from http://en.wikipedia.org/wiki/Regions_of_Venezuela
+"""
+
+#: A list of Venezuelan regions for use as `choices` in a formfield.
+REGION_CHOICES = (
+    ('501', _(u'Región Capital')),
+    ('502', _(u'Región Central')),
```
Comment (claudep) [inline localflavor/ve/ve_region.py]: Please remove the u'' prefixes. They are not necessary as you import unicode_literals anyway.

Comment (jackboot7) [inline localflavor/ve/ve_region.py]: Done. Updated PR.

**Comment by claudep** [comment]
Thanks for the contribution, but the patch will also need tests.

**Comment by jackboot7** [comment]
Will fix the errors and send a new one.

**Pull request closed without merging**